src/sprites/entities/player.py

```diff
@@ -98,6 +98,12 @@ def __init__(
         # check if the zoom is allowed
         self.zoom_allowed = False
 
+        # watch the player behaviour in achieving tutorial tasks
+        self.buyed_selled = False
```

Comment:
Can we change this to self.bought_sold?

Reply:
sure, no problems. I wasn't paying attention to that.